sdk/src/main/scala/com/horizen/validation/ConsensusValidator.scala

```diff
@@ -25,7 +25,7 @@ class ConsensusValidator extends HistoryBlockValidator with ScorexLogging {
       throw new IllegalArgumentException(s"Genesis block timestamp ${block.timestamp} is differ than expected timestamp from configuration ${history.params.sidechainGenesisBlockTimestamp}")
     }
 
-    val vrfSignIsNotCorrect = false //shall be implemented as VRF key will be used for block signing
+    val vrfSignIsNotCorrect = false //@TODO we should call verifyVfr and verifyForgerBox with consensusEpochInfo calculated from SC creation TX and a constant for nonce
```

Comment:
To validate genesis block we should check the following: 
 * verify existence of the sc creation tx with the same sc id in the first mainchain block reference 
 * verify that the sc block has been issued by the sc creator and moreover verify vrf output with vrf public key declared in the sc creation tx.